pactman/verifier/result.py

```diff
@@ -23,6 +27,9 @@ def start(self, interaction):
         super().start(interaction)
         log.info(f'Verifying {interaction}')
 
+    def warn(self, message):
+        log.warning(' ' + message)
```

Comment:
Why not  `log.warning(message)` ?

Reply:
The space is for nicer console output, all lines are indented under a "heading" line for the pact.